svelte.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
     plugins: ['es', 'jsdoc', 'svelte3', 'jest'],
     parserOptions: {
         sourceType: 'module',
-        ecmaVersion: 2015
+        ecmaVersion: 2020
```

Comment:
this will allow all features from es2020 which will also require updating the babel configs everywhere. Can't we just update the babel parser for now ?

Reply:
If you prefer, yes. I misunderstood your meaning in the other PR comments.